### PR TITLE
feat(android): Note about Room instrumentation

### PR DESCRIPTION
**Pull request by romtsn**
Add a note about Room integration to the Android wizard with a caveat that this is in beta

### CODE REVIEW

File: src/wizard/android/index.md [hunk 1]
```diff
@@ -116,6 +116,16 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
 
 Check out [the documentation](https://docs.sentry.io/platforms/android/performance/instrumentation/) to learn more about the API and automatic instrumentations.
 
+<Note>
+
+Living on the edge? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation)!
+  
+This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/proguard/#gradle) - make sure to use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room. Features in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (bruno-garcia) [inline src/wizard/android/index.md:123]:
```suggestion
This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/proguard/#gradle) - make sure to use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you simply set a tracing sample rate. Features in Beta are still work-in-progress and may have bugs. We recognize the irony.
```

**Review by bruno-garcia — APPROVED** [review]
Minor suggestions but otherwise LGTM.
Worth keeping in mind that the version is now fixed so we need to bump it here in follow up releases and eventually remove the note once we GA.

File: src/wizard/android/index.md [hunk 2]
```diff
@@ -116,6 +116,16 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
 
 Check out [the documentation](https://docs.sentry.io/platforms/android/performance/instrumentation/) to learn more about the API and automatic instrumentations.
 
+<Note>
+
+Living on the edge? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation).
+  
+This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/proguard/#gradle) - make sure to use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you simply set a tracing sample rate. Features in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (imatwawana) [inline src/wizard/android/index.md:123]:
```suggestion
This feature is available in the Beta release of the [Sentry Android Gradle plugin](https://docs.sentry.io/platforms/android/proguard/#gradle); you must use version `3.0.0-beta.1`. The `tracingInstrumentation` option is enabled by default, so Sentry automatically measures performance of the database queries done with Room if you set a tracing sample rate. Features in Beta are still a work-in-progress and may have bugs. We recognize the irony.
```

File: src/wizard/android/index.md [hunk 3]
```diff
@@ -116,6 +116,16 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
...
+
+Let us know if you have feedback via [GitHub issues](https://github.com/getsentry/sentry-android-gradle-plugin/issues).
```
Comment (imatwawana) [inline src/wizard/android/index.md:125]:
```suggestion
Let us know if you have feedback through [GitHub issues](https://github.com/getsentry/sentry-android-gradle-plugin/issues).
```

**Review by imatwawana — APPROVED** [review]
Minor style and grammar changes

**Comment by romtsn** [comment]
thanks @imatwawana

**Review by marandaneto — APPROVED** [review]

**Comment by romtsn** [comment]
Will merge it after resolving this issue https://github.com/getsentry/sentry-android-gradle-plugin/issues/228

File: src/wizard/android/index.md [hunk 4]
```diff
@@ -116,6 +116,16 @@ transaction.finish() // Mark the transaction as finished and send it to Sentry
 
 Check out [the documentation](https://docs.sentry.io/platforms/android/performance/instrumentation/) to learn more about the API and automatic instrumentations.
 
+<Note>
+
+Living on the edge? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation).
```
Comment (imatwawana) [inline src/wizard/android/index.md:121]: This idiom might not make sense for all users, so we should use something a bit more standard. 

```suggestion
Want to play with some new features? Try out our beta [Room performance integration](https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#sqlite-and-room-instrumentation).
```

**Review by imatwawana — COMMENTED** [review]
Suggested a new phrase for the first sentence of the note to avoid English idioms.

**Pull request merged**